dev/ProgressBar/ProgressBar.idl

```diff
@@ -0,0 +1,46 @@
+﻿namespace MU_XC_NAMESPACE
+{
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+unsealed runtimeclass ProgressBarTemplateSettings : Windows.UI.Xaml.DependencyObject
+{
+    ProgressBarTemplateSettings();
+
+    Double ContainerAnimationStartPosition;
+    static Windows.UI.Xaml.DependencyProperty ContainerAnimationStartPositionProperty{ get; };
+
+    Double ContainerAnimationEndPosition;
+    static Windows.UI.Xaml.DependencyProperty ContainerAnimationEndPositionProperty{ get; };
+
+    Windows.UI.Xaml.Media.RectangleGeometry ClipRect;
+    static Windows.UI.Xaml.DependencyProperty ClipRectProperty{ get; };
+
+    Double IndicatorLengthDelta;
```

Comment:
I don't see this being used anywhere, will it be in a subsequent change?

Reply:
Same as template settings question above

https://github.com/microsoft/microsoft-ui-xaml/pull/1453#discussion_r336699263